internal/factsengine/gatherers/corosyncconf.go

```diff
@@ -58,7 +58,9 @@ func (s *CorosyncConfGatherer) Gather(factsRequests []entities.FactRequest) ([]e
 		return nil, CorosyncConfFileError.Wrap(err.Error())
 	}
 
-	corosyncMap, err := corosyncConfToMap(corosyncConfile)
+	elementsToList := map[string]bool{"interface": true, "node": true}
```

Comment:
This is a common golang way to see if value is in a list. As golang doesn't have the `contains` or `in` functions, using a map is normal